rootx/src/rootx.cxx

```diff
@@ -472,6 +471,26 @@ int main(int argc, char **argv)
          PrintUsage(argv[0]);
          return 1;
       }
+      if (!strncmp(argv[i], "--version", 9) || !strncmp(argv[i], "-version", 8) || !strncmp(argv[i], "--v", 3) || !strncmp(argv[i], "-v", 2)){
```

Comment:
We should only look for `"--version"`, please.

Reply:
Sure removing others.